### PR TITLE
Fix Build Error

### PR DESCRIPTION
**Pull request by yogeshbdeshpande**

### CODE REVIEW

**Comment by yogeshbdeshpande** [comment]
@SteveLasker Can you please review this and merge please!

**Comment by SteveLasker** [comment]
> *** YAML syntax error: (<unknown>): mapping values are not allowed in this context at line 51 column 10

**Comment by yogeshbdeshpande** [comment]
> > *** YAML syntax error: (): mapping values are not allowed in this context at line 51 column 10

Yes, not sure why it is failing still investigating !

File: draft-ietf-scitt-software-use-cases.md [hunk 1]
```diff
@@ -47,9 +47,8 @@ normative:
 informative:
   RFC9019: SUIT-ARCH
   RFC9124: SUIT-IM
-  TUF:
-    title: >
-      The Update Framework Overview
+  TUF: TUF
```
Comment (SteveLasker) [inline draft-ietf-scitt-software-use-cases.md]:
```suggestion
  TUF:
    target: https://theupdateframework.io/overview/
    title: The Update Framework Overview
```

File: draft-ietf-scitt-software-use-cases.md [hunk 2]
```diff
@@ -47,9 +47,8 @@ normative:
...
+    title: The Update Framework Overview
```
Comment (SteveLasker) [inline draft-ietf-scitt-software-use-cases.md]:
```suggestion
```

File: draft-ietf-scitt-software-use-cases.md [hunk 3]
```diff
@@ -47,9 +47,8 @@ normative:
...
     target: https://theupdateframework.io/overview/
```
Comment (SteveLasker) [inline draft-ietf-scitt-software-use-cases.md:51]:
```suggestion
```

File: draft-ietf-scitt-software-use-cases.md [hunk 4]
```diff
@@ -329,7 +328,7 @@ Consumers and other interested parties of a firmware update ecosystem want to:
 
 There is no standardized way to:
 
-- provide an update framework that allows validation of authenticity of firmware revisions (in addition to existing approaches, such as {{-SUIT-ARCH}}, {{-SUIT-IM}}, or {{TUF}})
+- provide an update framework that allows validation of authenticity of firmware revisions (in addition to existing approaches, such as {{-SUIT-ARCH}}, {{-SUIT-IM}}, or {{-TUF}})
```
Comment (SteveLasker) [inline draft-ietf-scitt-software-use-cases.md]:
```suggestion
- provide an update framework that allows validation of authenticity of firmware revisions (in addition to existing approaches, such as {{-SUIT-ARCH}}, {{-SUIT-IM}}, or {{TUF}})
```

**Review by SteveLasker — APPROVED** [review]
kramdown fixes applied. It now builds.
LGTM

**Pull request merged**